test/e2e_node/gpus.go

```diff
@@ -151,6 +151,29 @@ var _ = framework.KubeDescribe("GPU [Serial]", func() {
 
 			By("Checking if the pod outputted Success to its logs")
 			framework.ExpectNoError(f.PodClient().MatchContainerOutput(podFailure.Name, podFailure.Name, "Success"))
+
+			By("checking allocatable gpus")
+			gpusAvailableBefore := getGPUsAvailable(f)
```

Comment:
`Available` here refers to `Capacity/Allocatable`. What you need I assume is the number of GPUs that are free right?